controllers/k8ssandra/datacenters.go

```diff
@@ -48,14 +48,20 @@ func (r *K8ssandraClusterReconciler) reconcileDatacenters(ctx context.Context, k
 		// its fields are pointers, and without the copy we could end of with shared
 		// references that would lead to unexpected and incorrect values.
 		dcConfig := cassandra.Coalesce(kc.Spec.Cassandra.DeepCopy(), dcTemplate.DeepCopy())
-		cassandra.ApplySystemReplication(dcConfig, *systemReplication)
+
+		cassandra.ApplyAuthSettings(dcConfig, kc.Spec.Auth)
+
+		if kc.Spec.Auth {
+			cassandra.ApplySystemReplication(dcConfig, *systemReplication)
```

Comment:
@jsanda in K8ssandra 1.x these settings are only applied if auth is enabled. Is that accurate?

Reply:
That is correct; however, I think it would be fine to call `cassandra.ApplySystemReplication` even when auth is disabled. The amount of extra data from the system_auth keyspace should be quite small. It would be one less thing to worry about if auth is initially disabled and then later enabled.